ci/scripts/documentation_checks.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+source ${SCRIPT_DIR}/common.sh
+
+set +e
+
+# Ignore symlinks
+DOCS_MD_FILES=$(find -P ${MORPHEUS_ROOT}/docs/source/ -type f -iname "*.md")
+DOCS_RST_FILES=$(find -P ${MORPHEUS_ROOT}/docs/source/ -type f -iname "*.rst")
+EXAMPLES_MD_FILES=$(find -P ${MORPHEUS_ROOT}/examples/ -type f -iname "*.md")
```

Comment:
Try using `git ls-files '*.rb'` to search for all files in repo which have the right extension to be more forward looking.